### ISSUE TITLE
Problemfeld: Politischer Arm für wen?

### ISSUE THREAD

**Issue by Bitsammelwanne** [sebulino/Strategie#19]
Als Partei benötigen wir gesellschaftliche Gruppierungen, die uns als "ihren" politischen Arm betrachten, bei denen es also fast schon selbstverständlich ist, pro Piraten zu sein und offen darüber geredet wird. Also so etwas wie Friedensbewegungen und Umweltbewegungen, die die Grünen hoch gebracht haben, oder wie die Netzgemeinde es mal bei uns war. So etwas sind Keimzellen, aus denen Face-to-Face Werbung für die jeweilige Partei gemacht wird.

Dazu müssen wir in diesem Bereich möglichst wenig Konkurrenz aus auseren Parteien haben, bzw. die Konkurrenz durch mehr Kompetenz klein bekommen. Es müssen Mulitiplikatoren der Gruppierungen eingebunden werden, und auch möglichst viel Sichtbarkeit dort erreicht werden und in diesem Bereich keine politischen Kompromisse gemacht werden (weshalb das nur in wenigen Bereichen gleichzeitig geht)

**Comment by KaitoCross** [comment]
Meiner Meinung nach sind Piraten stark mit der Netzgemeinde generell und mit der White Hat Hackercommunity verwurzelt (Chaos Computer Club)

**Comment by Annnnette** [comment]
Ja, diese Communities müssen wir pflegen. Dort gibt es viele ehemalige Wähler, die später von uns enttäuscht waren. Deren Vertrauen müssen wir zurückgewinnen.
